### DISCUSSION TITLE
I Can't Connect to my Wifi when I Run SEB

### DISCUSSION THREAD

**Discussion by nurdinsiregar** [SafeExamBrowser/seb-win-refactoring#388] [General]
I Can't Connect to my Wifi when I Run SEB, even though before I run SEB I was connected to my Wifi. 
![Before run SEB](https://user-images.githubusercontent.com/102930176/161453437-187bcac7-25b7-4ea1-9a1b-22259504574c.png)
![AFTER RUN SEB](https://user-images.githubusercontent.com/102930176/161453450-2f264f5c-238b-4b22-ba15-2d085d1f3d88.jpeg)

**Comment by dbuechel** [comment]
Could you please post the log files of SEB, to be found under `%LocalAppData%\SafeExamBrowser\Logs`?

**Reply by nurdinsiregar** [reply]
I use SEB 2.4.1 and I can't get the log file, But I can give you error message 
![Screenshot (1)](https://user-images.githubusercontent.com/102930176/161530983-ba228ab3-2a3b-45e6-8a84-41540f78f2bf.png)

**Reply by dbuechel** [reply]
SEB 2.x isn't supported anymore, please update to the latest official release version of SEB 3.x.

**Reply by nurdinsiregar** [reply]
Ok. Thanks
